IMC/chatbot/Chatbot.md

```diff
@@ -0,0 +1,170 @@
+# Covid-19 and ChatBots
+
+![img](images/1.jpg)
+
+## What is a Chatbot?
+
+A Chatbot is a software that you can “chat with.” You can find Chatbots in messaging apps such as Facebook Messenger, or on websites through pop-ups. They can also be integrated with “virtual assistants” for customer service or tech support. Google Assistant and Amazon Alexa are both great examples.
+
+With the rapid improvements in Natural Language Processing (NLP), it has never been easier to create a chatbot. If you pay  attention to the structures of modern websites, you will notice that most of them use an existing chatbot integrated with the web pages. These bots are considered smart chatbots as they have the ability to understand language structures. Although limited in general, they provide all the necessary components of a virtual assistant to help you navigate a website. Let's see how easy is it to create a chatbot!
+
+First, let’s look into why we might need a chatbot:
+
+- **Navigation:** You want to help a user navigate through a website.
+- **Information:** Search for news relevant to you.
+- **Customer Assistance:** Your business requires a lot of customer assistance and you want to automate responses to complaints that do not require urgent attention.
+- **Social-Automation:** Automate restaurant reservations.
+- **Curiosity:** You just want to explore the limits of computer intelligence (or trickery, see the image below ⬇️)
+
+![img](images/2.gif)
+
+## Why are Chatbots important for business?
+
+There are a massive amount of active users online these days. Chatbots make it much easier to offer customer service. They can interact with multiple users at the same time and work 24 hours a day. They provide users with helpful information to enhance customer experience while reducing the cost of traditional customer service, and prevent your team from always answering the same FAQs.
+
+As you can see, the advantages of a chatbot are far-reaching. Let's start creating our own from the place the researchers and pioneers of Natural language understanding (NLU) started. **Curiosity**.
```

Comment:
what do you mean by curiosity? are you referencing how they are curious later?

Reply:
I just removed it. It doesn't seem to make sense here.